### PR TITLE
[bug] ensure redis service has started and is running #94

### LINKED ISSUE #94: [bug] Server Error (500) on Fedora Systems
On fedora systems after installation, Server Error (500) is received when admin tries to login.

**Steps to reproduce**: 
1. Run anisble-openwisp2
2. After completion open browser and login with username & password `admin` on `/admin`.

### PR DESCRIPTION
**Pull request by atb00ker**
Ensure redis is running after installation and added CI test for it!

Fixes #94

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -7,15 +7,9 @@
...
-    - chmod +x ${PWD}/tests/test.sh
+    # run ci tests
+    - ${PWD}/tests/runtests.sh
 
-    # Run tests.
-    - ${PWD}/tests/test.sh
-
-    # Make sure OpenWISP is running.
-    - ./tests/runtests.sh
   dependencies: []
```
Comment (atb00ker) [inline .gitlab-ci.yml]: I am not sure if this line serves any purpose, does it?

Comment (hispanico) [reply .gitlab-ci.yml]: It's use to to disable artifact passing for this job.

For more details see https://docs.gitlab.com/ce/ci/yaml/

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -142,7 +136,6 @@ Debian10:
     distro: debian:10
     playbook: test.yml
   <<: *testing_script
-  allow_failure: true
```
Comment (nemesisdesign) [inline .gitlab-ci.yml]: why is this line removed? Is it related to the redis bug? If not I would leave it there and address it later if needed

File: .travis.yml [hunk 3]
```diff
@@ -25,23 +25,15 @@ env:
 
 matrix:
   allow_failures:
-    - env: distro=debian:10 playbook=test.yml
```
Comment (nemesisdesign) [inline .travis.yml]: why is this line removed? Is it related to the redis bug? If not I would leave it there and address it later if needed

**Comment by nemesisdesign** [comment]
Great, thank you @atb00ker :+1:

**Pull request merged**